### PR TITLE
chore(kafka-logger): support configuration `meta_refresh_interval` parameter

### LINKED ISSUE #6033: bug: kafka-logger, always log `not found broker` after kafka-logger kafka broker up.  until reload apisix.
### Issue description

always log `not found broker`  after kafka-logger kafka broker up.  until reload apisix.

### Environment

- apisix version (cmd: `apisix version`): 2.10.1
- OS (cmd: `uname -a`):Linux test 3.10.0-957.el7.x86_64  SMP Thu Nov 8 23:39:32 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
- OpenResty / Nginx version (cmd: `nginx -V` or `openresty -V`):
nginx version: openresty/1.19.3.2
built by gcc 4.8.5 20150623 (Red Hat 4.8.5-44) (GCC)
built with OpenSSL 1.1.1l  24 Aug 2021
TLS SNI support enabled
configure arguments: --prefix=/duitang/dist/sys/openresty/nginx --with-cc-opt='-O2 -DAPISIX_BASE_VER=0.0.0 -DNGX_LUA_ABORT_AT_PANIC -I/usr/local/openresty/zlib/include -I/usr/local/openresty/pcre/include -I/usr/local/openresty/openssl111/include' --add-module=../ngx_devel_kit-0.3.1 --add-module=../echo-nginx-module-0.62 --add-module=../xss-nginx-module-0.06 --add-module=../ngx_coolkit-0.2 --add-module=../set-misc-nginx-module-0.32 --add-module=../form-input-nginx-module-0.12 --add-module=../encrypted-session-nginx-module-0.08 --add-module=../srcache-nginx-module-0.32 --add-module=../ngx_lua-0.10.19 --add-module=../ngx_lua_upstream-0.07 --add-module=../headers-more-nginx-module-0.33 --add-module=../array-var-nginx-module-0.05 --add-module=../memc-nginx-module-0.19 --add-module=../redis2-nginx-module-0.15 --add-module=../redis-nginx-module-0.3.7 --add-module=../ngx_stream_lua-0.0.9 --with-ld-opt='-Wl,-rpath,/duitang/dist/sys/openresty/luajit/lib -Wl,-rpath,-L/usr/local/openresty/zlib/lib -L/usr/local/openresty/pcre/lib -L/usr/local/openresty/openssl111/lib -Wl,-rpath,/usr/local/openresty/zlib/lib:/usr/local/openresty/pcre/lib:/usr/local/openresty/openssl111/lib' --add-module=/duitang/dist/opt/builder/output/openresty-1.19.3.2/../ngx_multi_upstream_module --add-module=/duitang/dist/opt/builder/output/openresty-1.19.3.2/../apisix-nginx-module --add-module=/duitang/dist/opt/builder/output/openresty-1.19.3.2/../lua-var-nginx-module --with-poll_module --with-pcre-jit --with-stream --with-stream_ssl_module --with-stream_ssl_preread_module --with-http_v2_module --without-mail_pop3_module --without-mail_imap_module --without-mail_smtp_module --with-http_stub_status_module --with-http_realip_module --with-http_addition_module --with-http_auth_request_module --with-http_secure_link_module --with-http_random_index_module --with-http_gzip_static_module --with-http_sub_module --with-http_dav_module --with-http_flv_module --with-http_mp4_module --with-http_gunzip_module --with-threads --with-compat --with-stream --with-http_ssl_module
- etcd version, if have (cmd: run `curl http://127.0.0.1:9090/v1/server_info` to get the info from server-info API):{"version":"2.10.1","boot_time":1641439475,"etcd_version":"3.5.0","up_time":83420,"last_report_time":1641522886,"id":"8d53dda0-5ec2-44b6-872c-70f18b0397a8","hostname":"wvr-ngxweb-0-20.duitang.net"}
- apisix-dashboard version, if have:
- the plugin runner version, if the issue is about a plugin runner (cmd: depended on the kind of runner):
- luarocks version, if the issue is about installation (cmd: `luarocks --version`):

### Steps to reproduce

1. use kafka logger
2. make karka borker down
3. until apisix error log apprear like 'not found borker' logs.
4. make kafka borker up
5. apisix not auto recover

### Actual result

after kafka borker up , send logs. 

### Error log

 buffered messages send to kafka err: not found broker, retryable: true, ....

### Expected result

notings

### PR DESCRIPTION
**Pull request by nic-chen**
### Description

Fixes #6033

please refer to:
https://github.com/doujiang24/lua-resty-kafka/issues/70

[ci on centos7](https://github.com/apache/apisix/actions/runs/4061647503/jobs/6991872804#logs) is run in container, so we couldn't restart kafka broker in the test.

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [x] I have added tests corresponding to this change
- [x] I have updated the documentation to reflect this change
- [x] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: docs/en/latest/plugins/kafka-logger.md [hunk 1]
```diff
@@ -61,6 +61,7 @@ It might take some time to receive the log data. It will be automatically sent a
...
+| meta_refresh_interval | integer | False    | 30              | [1,...]               | `refresh_interval` parameter in [lua-resty-kafka](https://github.com/doujiang24/lua-resty-kafka) specifies the time to auto refresh the metadata, in seconds.                                                                                                                                                                                                                                                                                                           |
```
Comment (spacewander) [inline docs/en/latest/plugins/kafka-logger.md:64]:
```suggestion
| meta_refresh_interval | integer | optional        | 30              | [1,...]               | `refresh_interval` parameter in [lua-resty-kafka](https://github.com/doujiang24/lua-resty-kafka) specifies the time to auto refresh the metadata, in seconds.                                                                                                                                                                                                                                                                                                           |
```

File: t/plugin/kafka-logger2.t [hunk 2]
```diff
@@ -895,3 +895,108 @@ hello world
...
+            ngx.say(body)
+        }
+    }
+
+--- response_body
+passed
+
+
+
+=== TEST 23: get the route, confirm the meta_refresh_interval is set
```
Comment (spacewander) [inline t/plugin/kafka-logger2.t:945]: Why do we need this test? We don't have the logic to change meta_refresh_interval.

Comment (nic-chen) [reply t/plugin/kafka-logger2.t:945]: OK, let's remove it.

Comment (nic-chen) [reply t/plugin/kafka-logger2.t]: removed.

File: docs/en/latest/plugins/kafka-logger.md (hunk 1 above)
Comment (nic-chen) [reply docs/en/latest/plugins/kafka-logger.md]: fixed.

**Review by spacewander — APPROVED** [review]

**Review by soulbird — APPROVED** [review]

**Pull request merged**